nestkernel/connection_manager.cpp

```diff
@@ -1117,14 +1117,7 @@ nest::ConnectionManager::get_connections(
 
   if ( source == 0 and target == 0 )
   {
-#ifdef _OPENMP
-#pragma omp parallel
-    {
-      thread t = kernel().vp_manager.get_thread_id();
-#else
-    for ( thread t = 0; t < kernel().vp_manager.get_num_threads(); ++t )
-    {
-#endif
```

Comment:
The `for` loop can indeed go away. The reason it is there in the first place is that when we still supported PThreads, we had to prepare some thread-related data structures serially using the loop, or use OpenMP parallelism if that was enabled. No PThreads, no loop!